STAR-Delegation/Makefile

```diff
@@ -57,7 +60,6 @@ clean:
 	-rm -f $(draft)-[0-9][0-9].txt
 	-rm -f $(draft)-[0-9][0-9].md
 	-rm -f metadata.min.js
-	-rm -f $(art_svg)
```

Comment:
Are you sure? I think the SVG files *should* be removed on `make clean`.

Reply:
The SVGs rendering of the corresponding ascii-art file is created on the fly and inlined into the XML and HTML by kdrfc.  They don't exist on disk anymore.